accounts-db/src/buffered_reader.rs

```diff
@@ -449,7 +428,7 @@ mod tests {
         // First read 16 bytes to fill buffer
         let file_len_valid = 32;
         let default_min_read = 8;
-        let mut reader = BufferedReader::new(backing).with_file(&sample_file, file_len_valid);
+        let mut reader = BufferedReader::<16>::new().with_file(&sample_file, file_len_valid);
```

Comment:
Too bad we lose a bit of ensuring the test case size of `buffer_size` is the same as the struct generic param (`<16>`) here.

I think we get coverage by this line though:
```rust
assert_eq!(slice.len(), buffer_size);
```
Let me know if I'm wrong.